pyope/ope.py

```diff
@@ -169,3 +170,6 @@ def tape_gen(self, data):
             for bit in bits:
                 yield bit
 
+def generate_key():
```

Comment:
It would be useful to have a more generalized function which, for example, accepts key length as an optional argument. Also, is there a reason behind using `base64.urlsafe_b64encode`, and not the ordinary `base64.b64encode`?